cantools/db/formats/kcd.py

```diff
@@ -120,6 +121,8 @@ def _load_message_element(message, bus_name):
             is_extended_frame = (value == 'extended')
         else:
             LOGGER.debug("Ignoring unsupported message attribute '%s'.", key)
+            
+    length = message.attrib.get('length', 'auto')
```

Comment:
I suggest removing `, 'auto'` to make the length `None` is the length attribute is missing.